### ISSUE TITLE
[CI] builds failing due to @ibma/aat

### ISSUE THREAD

**Issue by tay1orjones** [IBM/carbon-addons-iot-react#1302]
### Describe the bug

PR builds began failing yesterday in Travis CI due to an issue with the external server that is used to validate the accessibility rules through `@ibma/aat`

Reached out to the team on internal slack, they've fixed it for us for now. They notified me that the package we're using is being deprecated and we need to migrate to a new package, `accessibility-checker`.

This morning it appears builds are passing, but we need to upgrade the package to avoid this in the future.
